db.go

```diff
@@ -73,7 +73,7 @@ func (u AuthUserType) CanRenameIdentity() bool {
 type UserStore interface {
 	Authenticate(identity, password string, authTypeCheck AuthCheck) error                  // Return nil error if the username and password are correct, otherwise one of ErrIdentityAuthNotFound or ErrInvalidPassword
 	SetPassword(userId UserId, password string, enforceTypeCheck PasswordEnforcement) error // This sets the password to a user account
-	SetConfig(passwordExpiry time.Duration) error
+	SetConfig(passwordExpiry time.Duration, enableAccountLocking bool, maxFailedPasswords int) error
```

Comment:
See comments below -- I don't think we need this change to the the SetConfig function